widgetssdk/src/main/java/com/glia/widgets/core/dialog/DialogController.java

```diff
@@ -49,10 +49,20 @@ public boolean isOverlayDialogShown() {
         return dialogsState instanceof DialogsState.OverlayPermissionsDialog;
     }
 
+    public boolean isExitingDialogShown() {
```

Comment:
Minor suggestion: I think `isExitQueueDialogShown()` is a little bit better name because "exiting" is very much close to "exciting". 🙂